qemu/tests/cfg/qemu_guest_agent.cfg

```diff
@@ -140,6 +140,8 @@
             time_service_start_cmd = net start w32time
             pause_time = 180
         - check_fstrim:
+            setsebool_cmd = "setsebool virt_qemu_ga_read_nonsecurity_files on"
```

Comment:
These configs are the same in every test case,could you take them out as a common config?